plugins/obs-vst/obs-vst.cpp

```diff
@@ -155,9 +166,7 @@ static void vst_save(void *data, obs_data_t *settings)
 	VSTPlugin *vstPlugin = (VSTPlugin *)data;
 	obs_data_set_string(settings, "chunk_data",
 			    vstPlugin->getChunk().c_str());
-	obs_data_set_string(
-		settings, "chunk_hash",
-		getFileMD5(vstPlugin->getEffectPath().c_str()).c_str());
+	obs_data_set_int(settings, "chunk_unique_id", vstPlugin->getUniqueID());
```

Comment:
I would advise to remove the old `chunk_hash` setting if a non-zero unique ID was found, effectively upgrading a VST configuration from being file-hash based to be unique ID-based.